src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    public Bulldozer() {
+        name = "Bulldozer";
+    }
```

Comment:
we can remove the constructor from the class and use "Bulldozer" directly in the methods

Reply:
removed